openlibrary/data/dump.py

```diff
@@ -220,6 +220,8 @@ def split_dump(dump_file=None, format="oldump_%s.txt"):
         "/type/list",
     )
     files = {}
+    files['misc'] = xopen(format % 'misc', 'wt')
```

Comment:
Calling this "other" since it's a catch-all
```suggestion
    files['other'] = xopen(format % 'other', 'wt')
```